app/src/main/java/chat/rocket/android/chatroom/presentation/ChatRoomPresenter.kt

```diff
@@ -587,6 +588,47 @@ class ChatRoomPresenter @Inject constructor(
         }
     }
 
+    fun uploadSharedFile(roomId: String, file: ShareHandler.SharedFile) {
```

Comment:
@Shailesh351 In this PR we have implemented compression for all image file uploads.  https://github.com/WideChat/Rocket.Chat.Android/commit/704fff0f6f4f4f24aacc7444b7df5307cb277d37

It looks like this PR does not include that for shared images.... can we add that?